packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -121,6 +125,16 @@ export class ApexTestRunner {
   }
 
   public async runApexTests(tests: string[], testRunType: TestRunType) {
+    const languageClientStatus = languageClientUtils.getStatus() as LanguageClientStatus;
```

Comment:
In the scenario where Java is not present or not the right version, the test sidebar will still allow the user to refresh or click the run test buttons. This will now surface an info message telling them why the tests are not present and how to fix the configuration.
![test_sidebar_message](https://user-images.githubusercontent.com/1041842/56084186-2e2ae300-5de4-11e9-81d3-b1abe7ee6f35.png)

Reply:
I think you can remove "more".